examples/config-generator/assets/ls1config.xml.template

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mardyn version="20100525">
+	<loglevel>ERROR</loglevel>
+	<refunits type="SI">
+		<length unit="nm">0.34</length>
+		<mass unit="u">39.948</mass>
+		<energy unit="K">120</energy>
+	</refunits>
+	<simulation type="MD">
+		<integrator type="Leapfrog">
+			<timestep unit="reduced">0.005</timestep>
+		</integrator>
+		<run>
+			<currenttime>0</currenttime>
+			<equilibration>
+				<steps>0</steps>
+			</equilibration>
+			<production>
+				<steps>15000</steps>
+			</production>
+		</run>
+		<ensemble type="NVT">
+			<temperature unit="reduced">1.10</temperature>
+			<domain type="box">
+				<lx>{md-size}</lx>
+				<ly>{md-size}</ly>
+				<lz>{md-size}</lz>
+			</domain>
+			<components>
+				<moleculetype id="1" name="Arbitrary">
+					<site type="LJ126" id="1" >
+						<coords>
+							<x>0.0</x>
+							<y>0.0</y>
+							<z>0.0</z>
+						</coords>
+						<mass>1.0</mass>
+						<sigma>1.0</sigma>
+						<epsilon>1.0</epsilon>
+					</site>
+				</moleculetype>
+			</components>
+			<phasespacepoint>
+				<generator name="MultiObjectGenerator">
+					<objectgenerator>
+						<filler type="GridFiller">
+							<lattice system="cubic" centering="face">
+								<vec id="a">
```

Comment:
as discussed, wherever there's {grid-filler-lattice}, this value can simply be 1, we can stop using the calculation since it's ignored if density is specified. Do we still want to keep this calculation?